### ISSUE TITLE
Test simulation set

### ISSUE THREAD

**Issue by tbarchyn** [IM3S/LDAR_Sim#14]
@tarcadius @KeeganShaw-GIS  Soroush

Noting this one down for action plan discussion:

1) need to build a default simulation that flexes as many muscles as LDAR sim has - so most of the code base is being run.

2) need to set random number seed to be constant. I'm reasonably certain this needs to be set in the code that runs on each process, not in ldar_sim_main.py. Or alternatively, run the test in serial. I'm certain this is pretty easy with numpy.

3) need to have the correct answer pre-calculated to know what the right answer is, so assert whether test passed or failed. With a set seed, there should be zero stochasticity and every test run should produce precisely the same result, except for maybe a few decimal places. In other words, this test simulation does not need to compare distributions or even run any replicates of any simulation. 

And, ideally this can be run with the github continuous integration hooks - but this could be a lot of work to set up - I've never set CI up on github? @KeeganShaw-GIS Soroush - do you have experience setting this up and is this workable?

Thoughts?

**Comment by tarcadius** [comment]
@KeeganShaw-GIS  I know you're super interested in this stuff and have a bit of experience with it? I certainly have minimal value to add on this topic!

**Comment by KeeganShaw-GIS** [comment]
@tbarchyn Yup sounds good, I changed this to CI Integration and added for potential release in V2.0.

**Comment by tarcadius** [comment]
@tbarchyn @KeeganShaw-GIS Updates on this? Plan for v2?
